Doc/faq/design.rst

```diff
@@ -710,7 +710,7 @@ Why don't generators support the with statement?
 For technical reasons, a generator used directly as a context manager
 would not work correctly.  When, as is most common, a generator is used as
 an iterator run to completion, no closing is needed.  When it is, wrap
-it as "contextlib.closing(generator)" in the 'with' statement.
+it as ``contextlib.closing(generator)`` in the :keyword:`with` statement.
```

Comment:
Just changed this to literal as a link would remove the 'generator' example

Reply:
Here you could use `` :func:`contextlib.closing(generator) <contextlib.closing>` ``.